### DISCUSSION TITLE
【灌水】六分街的那个录像店

### DISCUSSION THREAD

**Discussion by Luli-Chen** [share121/inter-knot#91] [其他]
![5b05568406624ad7abb8f81c6740b856_6923476599749336898](https://github.com/user-attachments/assets/bcab3a21-74d5-49a7-b5c8-a53ae7d39867)
好想去揉揉里面的邦布！

**Comment by koyki233** [comment]
我住十三分街有幸去过一次，邦布真的超duang，特别是那个叫伊埃斯的小邦布

**Comment by Luli-Chen** [comment]
真的？那我马上空出时间去溜一圈

**Comment by 132641515Zzy** [comment]
去六分街的录像带兄妹那，他们家的邦布乃是一绝

**Comment by koyki233** [comment]
他们家的录像带也很好看，连《星辉骑士》都有，真是爱了

**Comment by Nie7bai** [comment]
格莉丝小姐上大号说话
